core/src/toga/window.py

```diff
@@ -227,6 +247,17 @@ def __init__(
         self._closable = closable
         self._minimizable = minimizable
 
+        # Prime up the event handlers on the interface, as they might be
+        # called during the initialization of the native window class.
```

Comment:
On re-review, this raised a red flag for me. Handlers should not be firing as a result of initial conditions passed to the constructor. 

If the issue is that the code might result in the handler being invoked, and the handler need to exist, then the priming should be with a `None` value, with the "live" handler installed later. That ensures that `self.interface.on_something()` can *work*, but won't fire an actual handler until construction is complete.